### PR TITLE
Javascript/javascript2/week2

### PR DESCRIPTION
**Pull request by Divyajg**

### CODE REVIEW

File: javascript/javascript2/week2/js2week2.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
...88168, "running_times": 5700 }, { "title": "Just Married", "year": 2003, "rating": 5.5, "votes": 61009, "running_times": 5700 }, { "title": "Just My Luck", "year": 2006, "rating": 5.4, "votes": 47721, "running_times": 6180 }, { "title": "Just One of the Guys", "year": 1985, "rating": 6.5, "votes": 10976, "running_times": 6000 }, { "title": "Just Visiting", "year": 2001, "rating": 5.8, "votes": 16255, "running_times": 5280 }, { "title": "Just Wright", "year": 2010, "rating": 5.9, "votes": 11239, "running_times": 6000 }, { "title": "Justice League", "year": 2017, "rating": 7.2, "votes": 141370, "running_times": 7200 }, { "title": "Justin Bieber: Never Say Never", "year": 2011, "rating": 1.6, "votes": 74720, "running_times": 6900 }, { "title": "K-19: The Widowmaker", "year": 2002, "rating": 6.7, "votes": 52505, "running_times": 8280 }, { "title": "K-9", "year": 1989, "rating": 6, "votes": 27515, "running_times": 6060 }, { "title": "K-PAX", "year": 2001, "rating": 7.4, "votes": 160033, "running_times": 7200 }, { "title": "Kabhi Alvida Naa Kehna", "year": 2006, "rating": 6.1, "votes": 15518, "running_times": 11580 }, { "title": "Kabhi Haan Kabhi Naa", "year": 1994, "rating": 8, "votes":
```
Comment (esteficodes) [inline javascript/javascript2/week2/js2week2.js:73]: Great! Or we can also use or || (or), just for the sake of shortness, like in the previous comment.

const count = movies.filter((movie) => 
            (movie.title.toLowerCase().includes("surfer") || movie.title.toLowerCase().includes("alien") || movie.title.toLowerCase().includes("benjamin")));
console.log(count);

And maybe you can find a name for this function that makes it clearer what goes on inside of it? Something like selectedMovies or keywordMovies? 🤓

**Review by esteficodes — COMMENTED** [review]
Impressive job Divya! 💯  Just keep in mind details like arrow functions sintaxis (no use for return) and so. Well done!

File: javascript/javascript2/week2/js2week2.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Divyajg) [reply javascript/javascript2/week2/js2week2.js:52]: I tried using the above code, it was showing an error 'movies.foreach is not a function' :

[the remark this answers is not in the document]
Comment (esteficodes) [reply javascript/javascript2/week2/js2week2.js:52]: > I tried using the above code, it was showing an error 'movies.foreach is not a function' :

Oops, I think foreach is misspelled. It should be forEach (camel case)

[the remark this answers is not in the document]
Comment (Divyajg) [reply javascript/javascript2/week2/js2week2.js:23]: When I removed return, its not returning any value!

[the remark this answers is not in the document]
Comment (esteficodes) [reply javascript/javascript2/week2/js2week2.js:23]: Hmm, did you console.log the value? If you did, then I'll look into that. Leave your solution for now. It wasn't wrong.😃

**Comment by Divyajg** [comment]
@esteficodes : I have modified with all of your suggestions. Have a look and let me know if it is ready to merge! 
Thank you for your review, it helped a lot to understand the arrow functions more clearly and properly.

**Comment by esteficodes** [comment]
> @esteficodes : I have modified with all of your suggestions. Have a look and let me know if it is ready to merge!
> Thank you for your review, it helped a lot to understand the arrow functions more clearly and properly.

Thank you for your nice words, I'm glad to be of help. 🙏🏼😊

You might merge it whenever you are ready. My review was only a comment, I didn not request changes. 😊

**Comment by Divyajg** [comment]
Thank you.

**Pull request merged**